import-manual-mapping.html

```diff
@@ -85,22 +73,19 @@ <h2>Import - Manual Mapping</h2>
 
                             <sp-button-group>
                                 <sp-button id="detect-sections-button">Detect Sections</sp-button>
-                                <sp-button id="import-doimport-button">Import</sp-button>
+                                <sp-button id="save-transformation-button">Save</sp-button>
```

Comment:
Any chance (or need) for the user to inspect the transformation before/while saving?  Even in a confirmation dialog or popup?  Might help to educate the field about the concept?  Maybe we could link to our internal wiki to explain it for now?